frontend/src/css/cat-theme.css

```diff
@@ -149,3 +149,8 @@ Progess Steps
 .nav-pills .show > .nav-link {
   background-color: #00565e;
 }
+
+/* Notepad.jsx Text Area Focus Style */
+.text-area:focus {
```

Comment:
If this isn't overriding something in Aurora, can it go elsewhere? emib-tabs.css seems more appropriate.

Reply:
Fixed in commit #https://github.com/code-for-canada/project-thundercat/pull/113/commits/04ab846bf9cde5e890021bcc432571d4956a770b